src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.html

```diff
@@ -1,9 +1,29 @@
 <div *ngIf="exercise">
-    <h4 jhiTranslate="artemisApp.fileUploadAssessment.dashboard.heading" [translateValues]="{ exerciseTitle: exercise.title }"></h4>
-    <jhi-assessment-filters [submissions]="submissions" (filterChange)="updateFilteredSubmissions($event)"></jhi-assessment-filters>
+    <h4 [translateValues]="{ exerciseTitle: exercise.title }" jhiTranslate="artemisApp.fileUploadAssessment.dashboard.heading"></h4>
+    <jhi-assessment-filters (filterChange)="updateFilteredSubmissions($event)" [submissions]="submissions"></jhi-assessment-filters>
     <ng-container *ngIf="!busy"></ng-container>
     <jhi-assessment-warning [exercise]="exercise" [submissions]="filteredSubmissions"></jhi-assessment-warning>
-    <div class="table-responsive" *ngIf="submissions && submissions.length > 0; else emptyTable">
+    <h4>
+        <ng-container *ngIf="submissions && submissions.length === 0">
+            <small [ngClass]="'text-warning'">
+                {{ 'artemisApp.fileUploadAssessment.noSubmissions' | artemisTranslate: {} }}
```

Comment:
```suggestion
                {{ 'artemisApp.fileUploadAssessment.noSubmissions' | artemisTranslate }}
```

Reply:
Thanks! Solved in 2583cef5355323aad2b40177f34b236a4a54bf3d